reinforcement-learning/_posts/2018-12-17-tracto_td_learning.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: review
+title: "Tractography using reinforcement learning and adaptive-expanding graphs"
+tags: reinforcement-learning tractography
+author: "Antoine Théberge"
+cite:
+    authors: "Tingyi Wanyan, Lantao Liu, Eleftherios Garyfallidis"
+    title:   "Tractography using reinforcement learning and adaptive-expanding graphs"
+    venue:   "ISMRM 2015"
+pdf: "https://drive.google.com/open?id=1gYXIGYd6jRE0HCSvvx5NTRTBKUMLYBrb"
+---
+
+## Introduction
+
+Currently, three types of tractography algorithm exist: Deterministic, probabilistic and global algoritms. The authors propose a novel approach based on reinforcement-learning.
```

Comment:
*(...) three types of tractography algorithm exist: Deterministic, (...)* to *(...) three types of tractography algorithms exist: deterministic, (...)*

Reply:
Fixed !